test/users/engin/compiler/parentheseslessRecConst.chpl

```diff
@@ -0,0 +1,20 @@
+record independentRecord { var field = 5;} 
+
+var ir = new independentRecord;
+writeln(ir.field);
+//above lines compile with/without --verify
```

Comment:
I don't think we support defining parenthesesless constructors, and the auto-generated ones definitely aren't parenthesesless.